cfgov/v1/util/js_routes.py

```diff
@@ -0,0 +1,78 @@
+## Page Specific JS Routes ##
+import os, shutil
+from django.conf import settings
+from . import util
+
+MOLECULE = 'molecules'
+ORGANISM = 'organisms'
+
+LOOKUP_TABLE = {
+    'Expandable': MOLECULE,
+    'ExpandableGroup': ORGANISM
+}
+
+
+def create_route(data):
+    type_found = False
+    elements = data['elements']
+    content = "'use strict';"
+
+    dir_depth = data['slug'].count('/')
+    path_depth = ''
+    for x in range(0, dir_depth):
+        path_depth += '../'
+
+    for idx, element in enumerate(elements):
+        class_name = util.to_camel_case(element)
+        type = LOOKUP_TABLE[class_name]
```

Comment:
This is erroring because `class_name` is getting the name `ExpandablesGroup` instead of `ExpandableGroup`. It should be singular to align with design wiki.